src/app/db/heart-beat-db.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const MongoClient = require('mongodb').MongoClient;
+const url = 'mongodb://localhost:27017/heartbeat';
+
+const checkDbHealth = function(res) {
+  MongoClient.connect(url, function(err, db) {
+    if (err === null) {
+      let collection = db.collection('heartbeat');
+      collection.find({}).toArray(function(err, docs) {
+        if (err === null && docs && docs.length>0) {
+          res.json('{"status": "ok", "database": "ok"}');
```

Comment:
`res.json` accepts a JavaScript object as well and it will stringify it

`res.json({status: 'ok', database: 'ok})`

Reply:
put that object into a const variable